src/api/apiData.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+import columnWidths from '../selectors/columnWidths';
+import shallowEqualSelector from '../helper/shallowEqualSelector';
+
+const getApiDataSelector = () =>
+  shallowEqualSelector(
```

Comment:
This method returns the public data API that we expose to users.
The value is computed through a selector for efficiency.